core/src/main/scala/org/genivi/sota/core/WebService.scala

```diff
@@ -106,6 +110,36 @@ class VehiclesResource(db: Database, rviClient: RviClient)
         rviClient.sendMessage(s"genivi.org/vin/${vin}/sota/getpackages", io.circe.Json.Empty, ttl())
         // TODO: Confirm getpackages in progress to vehicle?
         complete(NoContent)
+      } ~
+      pathPrefix("updates") {
+        (pathEnd & post) {
+          entity(as[List[Package.Id]]) { ids ⇒
+            val f = new InstalledPackagesUpdate(db, resolverClient)
```

Comment:
It doesn't need to be a class, but it encapsulates that state. db and resolver client. We'd need to pass this around all the time if we don't have an instance to hold it.